external/easylogging++/easylogging++.h

```diff
@@ -1211,6 +1211,8 @@ class OS : base::StaticClass {
   /// @detail For android systems this is device name with its manufacturer and model seperated by hyphen
   static std::string currentHost(void);
   /// @brief Whether or not terminal supports colors
+  static bool termSupportsColor(std::string& term);
+  /// @brief Whether or not terminal supports colors
```

Comment:
Perhaps you can clarify this to something along the lines of "the terminal attached to this environment", and change the previous to "a terminal with this name"